flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/reconciler/deployment/AbstractFlinkResourceReconciler.java

```diff
@@ -174,6 +182,32 @@ public void reconcile(FlinkResourceContext<CR> ctx) throws Exception {
         }
     }
 
+    private void scaling(FlinkResourceContext<CR> ctx) throws Exception {
+        KubernetesJobAutoScalerContext autoScalerContext = ctx.getJobAutoScalerContext();
+
+        if (autoscalerDisabled(ctx)) {
+            autoScalerContext.getConfiguration().set(AUTOSCALER_ENABLED, false);
+            resourceScaler.scale(autoScalerContext);
+            return;
+        }
+        if (waitingForRunning(ctx.getResource().getStatus())) {
+            LOG.info("Autoscaler is waiting for stable, running state");
+            resourceScaler.cleanup(autoScalerContext.getJobKey());
+            return;
+        }
+        resourceScaler.scale(autoScalerContext);
```

Comment:
```suggestion
        if (autoscalerDisabled(ctx)) {
            autoScalerContext.getConfiguration().set(AUTOSCALER_ENABLED, false);
            resourceScaler.scale(autoScalerContext);
        } else if (waitingForRunning(ctx.getResource().getStatus())) {
            LOG.info("Autoscaler is waiting for stable, running state");
            resourceScaler.cleanup(autoScalerContext.getJobKey());
        } else {
            resourceScaler.scale(autoScalerContext);
        }
```

Much easier to read for me.